stock_account_quantity_history_location/wizards/stock_quantity_history.py

```diff
@@ -29,10 +37,27 @@ def open_at_date(self):
             )
             action["context"] = ctx
             if self.env.context.get("active_model") == "stock.valuation.layer":
+                # _domain_quant, domain_in, domain_out = (
+                #     self.env["product.product"]
+                #     .with_context(ctx)._get_domain_locations()
+                # )
+                # svl_domain = expression.OR(
+                #   [convert_domain(domain_in), convert_domain(domain_out)]
```

Comment:
This was the original approach. When converting that domain to filter the related moves, the performance drops dramatically.